bindings/pyroot_experimental/pythonizations/CMakeLists.txt

```diff
@@ -70,58 +63,62 @@ ROOT_STANDARD_LIBRARY_PACKAGE(ROOTPython
     src/TDirectoryPyz.cxx
     src/TFilePyz.cxx
     src/TMemoryRegulator.cxx
-    src/TPyArg.cxx
-    src/TPyClassGenerator.cxx
-    src/TPyReturn.cxx
-    src/TPython.cxx
     src/TObjectPyz.cxx
     src/TTreePyz.cxx
     src/PyzCppHelpers.cxx
     src/PyzPythonHelpers.cxx
     src/CPPInstancePyz.cxx
     ${PYROOT_EXTRA_SOURCE}
-  DICTIONARY_OPTIONS
-    -writeEmptyRootPCM
-  DEPENDENCIES
-    Core
-    Tree
-    cppyy
 )
 
 set(ROOTDirName python/ROOT)
-set(JupyROOTDirName python/JupyROOT)
-set(JsMVADirName python/JsMVA)
+set(Pyz_headers inc/Pythonizations)
```

Comment:
What is the reason for moving all the internal headers from `src` to `inc/Pythonizations` ?